vulture_os/toolkit/api_parser/sentinel_one_identity_graph/sentinel_one_identity_graph.py

```diff
@@ -231,9 +255,68 @@ def format_log(self, log: dict) -> str:
                     tactics.append(attack.get("tactic"))
                     techniques.append(attack.get("technique"))
 
-        log["indicators"] = {}
-        log["indicators"]["tactics"] = tactics
-        log["indicators"]["techniques"] = techniques
+        log["mitre_technique_ids"] = list(set([x["uid"] for x in techniques]))
+        log["mitre_technique_names"] = list(set([x["name"] for x in techniques]))
+        log["mitre_tactic_ids"] = list(set([x["uid"] for x in tactics]))
+        log["mitre_tactic_names"] = list(set([x["name"] for x in tactics]))
+
+        # Comments
+        if log.get('noteExists'):
+            comments = []
+            for comment in self.get_itdr_alert_comments(log.get("id")):
+                name = comment.get('author', {}).get('fullName', "")
+                email = comment.get('author', {}).get('email', "")
+                date = comment.get('createdAt').replace('Z', '+00:00')
```

Comment:
Warning, if comment.get() returns None, it will raise an error with the .replace()